build/eslint/xfail.list

```diff
@@ -1,11 +1,237 @@
+apps/clock/test/unit/utils_test.js
+apps/communications/contacts/js/bootstrap.js
+apps/communications/contacts/js/cache.js
+apps/communications/contacts/js/contacts_matching_ui.js
+apps/communications/contacts/js/views/list.js
+apps/communications/contacts/test/unit/contacts_action_menu_test.js
+apps/communications/contacts/test/unit/contacts_buttons_test.js
+apps/communications/contacts/test/unit/contacts_matching_ui_test.js
+apps/communications/contacts/test/unit/contacts_tag_test.js
+apps/communications/contacts/test/unit/export/generic_export_test.js
+apps/communications/contacts/test/unit/export/sd_test.js
+apps/communications/contacts/test/unit/image_loader_test.js
+apps/communications/contacts/test/unit/import/import_contacts_test.js
+apps/communications/contacts/test/unit/import/import_test.js
+apps/communications/contacts/test/unit/link_test.js
+apps/communications/contacts/test/unit/mock_multiple_select_dom.js
+apps/communications/contacts/test/unit/navigation_test.js
+apps/communications/contacts/test/unit/search_test.js
+apps/communications/contacts/test/unit/utilities/system_banner_test.js
+apps/communications/contacts/test/unit/views/details_test.js
+apps/communications/contacts/test/unit/views/form_test.js
+apps/communications/contacts/test/unit/views/ice_settings_test.js
+apps/communications/contacts/test/unit/views/ice_test.js
+apps/communications/contacts/test/unit/views/list_test.js
+apps/communications/contacts/test/unit/views/multiple_select_test.js
+apps/communications/contacts/test/unit/views/settings_test.js
+apps/communications/contacts/test/unit/webrtc-client/webrtc_client_test.js
+apps/communications/dialer/js/suggestion_bar.js
+apps/communications/dialer/test/unit/call_info_test.js
+apps/communications/dialer/test/unit/call_log_test.js
+apps/communications/dialer/test/unit/suggestion_bar_test.js
+apps/costcontrol/js/settings/settings.js
+apps/costcontrol/test/unit/common_test.js
+apps/costcontrol/test/unit/formatting_test.js
 apps/email/build/email.build.js
 apps/email/build/gelam_worker.build.js
 apps/email/build/main-frame-setup.build.js
+apps/email/js/ext/ext/bleach.js/index.js
+apps/email/js/html_cache_restore.js
+apps/email/js/template.js
+apps/email/js/tmpl.js
+apps/email/test/unit/message_reader_card_test.js
+apps/emergency-call/js/keypad.js
+apps/emergency-call/test/unit/ice_contacts_test.js
+apps/fm/js/fm.js
+apps/ftu/js/ui.js
+apps/ftu/js/wifi.js
+apps/ftu/test/unit/language_test.js
+apps/ftu/test/unit/navigation_test.js
+apps/keyboard/build/settings-config.js
+apps/keyboard/js/imes/jspinyin/tests/init.js
+apps/keyboard/js/views/emoji_key_view.js
+apps/keyboard/js/views/key_view.js
 apps/keyboard/test/unit/keyboard/layout_normalizer_test.js
+apps/keyboard/test/unit/settings/layout_item_view_test.js
+apps/music/js/utils.js
+apps/privacy-panel/test/unit/ala/define_custom_location_test.js
+apps/ringtones/js/tone_list.js
+apps/settings/js/firefox_accounts/panel.js
 apps/settings/js/modules/addon_manager.js
```

Comment:
Those were the files that failed after taking the jshintignore into account.
The updated list is without the taking the jshintignore files into account (i.e. greater coverage).